images/kubekins-e2e/Dockerfile

```diff
@@ -43,6 +43,8 @@ RUN apt-get update && \
 # preinstall this for kops tests xref kubetest prepareAws(...)
 # TODO(krzyzacy,justisb,chrislovecnm): remove this
 RUN pip install awscli
+# Allow generate junit.xml in python
+RUN pip install junit-xml
```

Comment:
why do we need this?

Reply:
when we replace this image I'd really prefer we don't do this.

if we _do_ do this, it shouldn't be _another_ layer.